pkg/web_app/lib/src/tabs.dart

```diff
@@ -38,15 +38,21 @@ void _setEventsForTabs() {
 
 /// change the tab based on URL hash
 void _changeTabOnUrlHash() {
-  if (tabRoot == null) return;
+  if (_headerRoot == null) return;
   String hash = window.location.hash ?? '';
   if (hash.startsWith('#')) {
     hash = hash.substring(1);
   }
   // Navigating back to a non-hashed package page will result an empty hash.
-  // Displaying the default tab: readme.
+  // Displaying the first tab (with content) by default.
   if (hash.isEmpty) {
-    changeTab('-readme-tab-');
+    final active = _contentRoot.children.firstWhere(
```

Comment:
Have you considered `_contentRoot.querySelector('.-active')`, see:
https://api.dartlang.org/stable/2.4.0/dart-html/Element/querySelector.html

It should return just one element :)

Just a nit..

Reply:
In this case it must be one if its direct children. I haven't checked if `>.-active` would work too, but will do (I'll need to update parts of this as tabs will change a bit more).